executor/aggfuncs/window_func_test.go

```diff
@@ -207,3 +207,17 @@ func (s *testSuite) TestMemRank(c *C) {
 		s.testWindowAggMemFunc(c, test)
 	}
 }
+
+func (s *testSuite) TestMemPercentRank(c *C) {
```

Comment:
IMO, we could move this to `func_percent_rank_test.go`. Note that don't forget the open source license for the new file.